i18n/de.yaml

```diff
@@ -62,6 +62,13 @@
 - id: android_keyrecovery_contact_us
   translation: "Wenn du keine E-Mail bekommen hast, <a class=\"text-primary no-underline hover:underline\" href=\"mailto:support@cryptomator.org\">kontaktiere uns</a>."
 
+- id: android_fdroid_title
+  translation: "App mittels F-Droid herunterladen"
+- id: android_fdroid_repo
+  translation: "Folgenden <a class=\"text-primary no-underline hover:underline\" href=\"https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD\">Link</a> öffnen oder den QR-Code scannen um unser Repository hinzuzufügen."
```

Comment:
```suggestion
  translation: "Folgenden <a class=\"text-primary no-underline hover:underline\" href=\"https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD\">Link</a> öffnen oder den QR-Code scannen, um unser Repository hinzuzufügen."
```

What about `fdroidrepos://`?

Reply:
Custom scheme aren't nice handled, rechecked it using a few more apps and the result is not good, e.g.:
<img src="https://user-images.githubusercontent.com/1786772/115222796-666c0700-a10b-11eb-8566-85bbe68a7096.png"  height="600" />

Selecting or using this URL leads to https://cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD in F-Droid which throws an error...

In my opinion, `fdroidrepos://` creates more potential problems for us than the fact that https://static.cryptomator.org/android/fdroid/repo?fingerprint=F7C3EC3B0D588D3CB52983E9EB1A7421C93D4339A286398E71D7B651E8D8ECDD cannot be opened in the browser on the desktop.